spec/factories/access_requests.rb

```diff
@@ -22,8 +22,24 @@
     email_address { Faker::Internet.email }
     first_name { Faker::Name.first_name }
     last_name { Faker::Name.last_name }
-    organisation { create(:organisation) }
+    organisation { requester.organisations.first.name }
```

Comment:
is meant to be `name` and not `organisation`?

Reply:
You are correct. Done. Thanks.